### DISCUSSION TITLE
Belajar

### DISCUSSION THREAD

**Discussion by BaeQueen** [deaafrizal/forum#81] [The Lounge]
Hallo kawan2 semua, Mau tanya kekurangan dan kelebihan punya website film apa ya?

**Comment by seporang** [comment]
ini ceritanya mau buat apa bisnis?

**Reply by BaeQueen** [reply]
Iya,rencananya

**Comment by nicolas-aditya-nugroho** [comment]
Bang cara buat kalender menggunakan codingan html, css, dan js dong bang 😌🙏
